package.json

```diff
@@ -41,7 +41,7 @@
     "indent-string": "^3.1.0",
     "inline-critical": "^3.0.0",
     "lodash": "^4.17.4",
-    "meow": "^3.7.0",
+    "meow": "^4.0.0",
```

Comment:
Meow v4 has breaking changes. You need to adapt to those: https://github.com/sindresorhus/meow/releases/tag/v4.0.0 (Note how specifying `minimist` options changed)

Reply:
This needs to be changed: https://github.com/addyosmani/critical/blob/2f332dc8986e31b820b33f2e2cd98b4816b3d81a/cli.js#L43-L57

Reply:
Thanks. I haven't even got so far, because tests fail anyway...